use_cases/scripts/fix_training_xmls.py

```diff
@@ -0,0 +1,76 @@
+from argparse import ArgumentError
```

Comment:
Apologies for the delay in this review. Could you add a module doc-string explaining the use of this script please.

Reply:
This script was for making a small change to the training XMLs that was causing an error. Future users shouldn't need to use the script. Maybe it would be better to just remove the script from git?

Reply:
We have a place in RAVEN for conversion scripts that is meant to provide means to stay current with input file changes; is there compatibility changes in this script that isn't reflected in those? See `raven/scripts/conversionScripts`.

Reply:
Sorry, actually diffing the XML files, I think I was able to resolve whatever issue I had been having and was able to revert the files to how they were. No compatibility issues and no changes to the files. So all the script in question really does now is let you change the verbosity setting in all of the RAVEN XML files.

Good to know about those conversion scripts though. I didn't know those existed.

Reply:
@dylanjm Also there are a lot of PNG files in the 2020_12 use case. Should we remove those from the repo as well? Or were you advocating adding PNGs to a .gitignore but not actually removing the files?

Reply:
Oh yeah, I forgot about those. I meant for those PNGs to never be merged into this FORCE repo from the previous repo they were in. Ideally I would prefer to remove all (past/future) PNGs from the repo, but am open to suggestions.

Reply:
As long as the figures can be reproduced through the RAVEN training files or other  scripts, I see no harm in removing the PNGs.

Reply:
Let's do that then.